File: mysql/fsharp/delete/mysql_delete.fs
// -------------------------------------------------------------------
//
//	mysql_delete.fs
//
//					Jun/11/2013
//
// -------------------------------------------------------------------
open System
open System.Data
open MySql.Data.MySqlClient

// -------------------------------------------------------------------
[<EntryPoint>]
let main (args : string[] ) =
  printfn "*** 開始 ***"
  let id_in = args.[0]

  let conn = new MySqlConnection "userid=scott;password=tiger;database=city;Host=host_mysql"
  conn.Open()

  let sql_str = "delete from cities where ID = '" + id_in + "'"
  printfn "%s" sql_str
  let cmd = new MySqlCommand (sql_str, conn )
  let rowsAffected = cmd.ExecuteNonQuery()

  printfn "rowsAffected = %d" rowsAffected

  conn.Close()
  printfn "*** 終了 ***"

  0

// -------------------------------------------------------------------

File: mysql/csharp/delete/mysql_delete.cs
// -------------------------------------------------------------------
/*
	delete/mysql_delete.cs

					Sep/17/2011


*/
// -------------------------------------------------------------------
using	System;
using	System.Data;
using	System.Text;
using	MySql.Data.MySqlClient;

// -------------------------------------------------------------------
class mysql_delete
{
// -------------------------------------------------------------------
static void Main (string[] args)
{
	Console.WriteLine ("*** 開始 ***");

	string	id_in = args[0];

	string server="host_mysql";
	string str_db="city";

	string str_connect = "Server=" + server +
	";User Id=scott;Password=tiger;" +
		"Database=" + str_db + ";";
 
	MySqlConnection connection = new MySqlConnection (str_connect);
	connection.Open ();

	mysql_manipulate.mysql_delete_proc (connection,id_in);

	connection.Close ();

	Console.WriteLine ("*** 終了 ***");
}

// -------------------------------------------------------------------
}

// -------------------------------------------------------------------
